<launch>
    <arg name="message_display" default="screen" doc="output type"/>
    <arg name="main_hz" default="1000" doc="main loop herz"/>
    <arg name="PUB_SEC_PERIOD" default="1" doc="PUB_SEC_PERIOD"/>
    
    <node pkg="faduino" type="dummy" name="dummy" output="$(arg message_display)">
        <param name="main_hz" value="$(arg main_hz)"/>
        <param name="PUB_SEC_PERIOD" value="$(arg PUB_SEC_PERIOD)"/>
    </node>
</launch>
